### PR TITLE
feat: instrument vertx-web-openapi and vertx-web-openapi-router

### PR DESCRIPTION
**Pull request by pk-work**
see #176

### CODE REVIEW

File: vertx-lang-kotlin/pom.xml [hunk 1]
```diff
@@ -203,6 +203,16 @@
...
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web-openapi</artifactId>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web-openapi-router</artifactId>
```
Comment (tsegismont) [inline vertx-lang-kotlin/pom.xml:213]: Same here, you need to add the artifact ids to the list in the maven dependency plugin config.

**Review by tsegismont — COMMENTED** [review]
Thanks

File: vertx-lang-kotlin/pom.xml (hunk 1 above)
Comment (pk-work) [reply vertx-lang-kotlin/pom.xml:213]: done

**Review by tsegismont — APPROVED** [review]

**Comment by tsegismont** [comment]
LGTM

**Pull request merged**